R/bookdown_to_leanpub.R

```diff
@@ -237,6 +244,10 @@ bookdown_to_leanpub <- function(path = ".",
     message("Copying bib files")
   }
   copy_bib(path, output_dir = output_dir)
+  copy_quizzes(path, output_dir = output_dir)
```

Comment:
Just so its parallel to the others, can we put message first, then function?

Reply:
yup!

Reply:
Yup that's it!